packages/webapp/src/pages/members/index.tsx

```diff
@@ -53,7 +53,6 @@ export const MembersPage = (props: Props) => {
 
     const { data: memberStats } = useQuery({
         ...queryMembersStats,
-        keepPreviousData: true,
```

Comment:
@brandonfancher @sparkplug0025 I believe this `keepPreviousData` isn't needed either, right? This wasn't my addition, which is why I'm asking. I don't want to undo something that was intentional...

Reply:
The `keepPreviousData` helps by caching pagination: https://react-query.tanstack.com/guides/paginated-queries#better-paginated-queries-with-keeppreviousdata

I don't see a reason to remove it... Did I miss something?

Reply:
Yeah, if it's helping smooth pagination for a pagination-related query, it's relevant. (And I believe pagination has a dependency on this query here.)

Now, if we wanted to, we _could_ switch this out for the new named query--`useMemberStats()`--and keep the `keepPreviousData` flag on that one.